Most studies proposing new strategies to manage and treat infections have been largely focused on identifying druggable targets that can inhibit a pathogen's growth when it is the single cause of infection. In vivo, however, infections can be caused by multiple species. This is important to take into account when attempting to develop or use current antibacterials since their efficacy can change significantly between single and co-infections. In this study, we used genome-scale metabolic models (GEMs) to interrogate the growth capabilities of Vibrio cholerae(V. cholerae) in single and co-infections with enterotoxigenicE. coli(ETEC). Co-infection model predictions showed that V. choleraegrowth capabilities are enhanced in presence of ETEC relative to V. choleraesingle-infection, through cross-fed metabolites made available to V. choleraeby ETEC. In vitro, co-cultures of the two enteric pathogens further confirmed model predictions showing an increased growth of V. choleraein co-culture relative to V. choleraesingle-cultures while ETEC growth was suppressed. Dual RNAseq analysis of the co-cultures also confirmed that the transcriptome of V. choleraeis distinct during co-infection compared to single infection scenarios where processes related to metabolism were significantly perturbed. Further, in silicogene-knock out simulations uncovered discrepancies in gene essentiality for V. choleraegrowth between single and co-infections. Integrative model-guided analysis thus identified druggable targets that would be critical for V. choleraegrowth in both single and co-infections, thus designing inhibitors against those targets would provide a broader spectrum coverage against cholera infections.
